### ISSUE TITLE
A11y_GradeB_VSCode_Keyboard shortcut reads words together - Blind: Arrow key navigation to row Find the binding keys and  "when" cell data are read together resulting in a word " CTRL + FeditorFocus  instead of CTRL + F editorFocus"

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#182490]
**GitHubTags**:#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#DesktopApp;#BM-VisualStudioCode-Win32-Nov2021;#NVDA;#Win32;#Win11;#AINotInScope;

## Environment Details:​
Application Name: Visual Studio Code
Version: 1.79.0-insider (user setup)
OS build 25300.1000
JAWS 2023.2303.144

## Repro Steps:

1. Launch Visual Studio Code insiders
2. Navigate to settings and press enter
3. tab to keyboard shortcut and press enter
4. start JAWS
5. tab to the row with item as find

## Actual:
Arrow key navigation to row Find the binding keys and  "when" cell data are read together resulting in a word " CTRL + FeditorFocus  instead of CTRL + F editorFocus"

## Expected:
Arrow key navigation to row Find the binding keys and  "when" cell data  should be read as CTRL + F editorFocus"

## User Impact:
screen reader users will not know the data 

## Attachment
[Keyboard shortcut reads words together-JAWS.webm](https://github.com/microsoft/vscode/assets/24580570/a24db630-c042-4d12-a18a-215e6f8d2e1d)

**Comment by fsteffi** [comment]
#closed;
Version: 1.79.0-insider (user setup)
Commit: c3867859cf3e1a73f3ddce5ad36b3791b7883d88
Date: 2023-06-01T05:24:05.541Z
Electron: 22.5.4
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25370

[Video 6-01 at 16.53.webm](https://github.com/microsoft/vscode/assets/24580570/b7c6a1a4-c65d-423b-acc6-8ae13e7c19fd)

**Comment by fsteffi** [comment]
#closed;
Version: 1.79.0-insider (user setup)
Commit: c3867859cf3e1a73f3ddce5ad36b3791b7883d88
Date: 2023-06-01T05:24:05.541Z
Electron: 22.5.4
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25370

[Video 6-01 at 16.53.webm](https://github.com/microsoft/vscode/assets/24580570/b7c6a1a4-c65d-423b-acc6-8ae13e7c19fd)
